docs/src/main/asciidoc/Developer_Guide.adoc

```diff
@@ -49,7 +49,7 @@ include::_developer-guide/JTA_Reference.adoc[]
 
 include::_developer-guide/JNDI_Reference.adoc[]
 
-include::_developer-guide/JAX-RS_Reference_Guide.adoc[]
+include::_developer-guide/Jakarta_RESTful_Web_Services_Reference_Guide.adoc[]
```

Comment:
JAX-RS appears in Jakarta_RESTful_Web_Services_Reference_Guide.adoc three times.

Reply:
following this document - https://jakarta.ee/legal/acronym_guidelines/